docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/src/entrypoints/specialReport/SpecialReport.scss

```diff
@@ -26,7 +24,7 @@
 
     @include break(medium) {
       max-height: 384px;
-      height: 37.5w;
+      height: 37.5 w;
```

Comment:
this is probably supposed to be 37.5 vw

Reply:
Will modify and check locally

Reply:
Turns out the  line was not being applied in browsers and is actually not necessary so I removed it entirely.